pyproject.toml

```diff
@@ -17,13 +17,15 @@ classifiers = [
 description = "File handler utilizing TUS and MinIO with DRS-Filer integration."
 license = "Apache-2.0"
 maintainers = ["ELIXIR Cloud AAI <cloud-service@elixir-europe.org>"]
-name = "tus_storagehandler"
+name = "Tus-Storage-Handler"
```

Comment:
According to convention, package names should be all lowercase, with underscores possible though discouraged. You can leave it for now, but when you change the name in the next PR, please pick one with all lower case, perhaps `cloud_storage_handler`.